packages/webapp/src/components/Navigation/index.jsx

```diff
@@ -51,6 +52,7 @@ export default function PureNavigation({
               open={showNotificationSpotlight}
               onFinish={resetSpotlight}
             />
+            <ReleaseBadgeHandler />
```

Comment:
Would this be more appropriately in App? The is Farm selected is handy here but it doesn't necessarily seem navigation related.

Reply:
Can it be done from App and still maintain dependence on `isFarmSelected`? I wrote it in the old `<NavBar />` originally (back when there was a demarcation between farmSelected and not farmSelected there), not knowing that component was going to be refactored. I'm not at all particular about where it's placed, as long as it doesn't appear until the login is completed -- do you know of a better place with that logic?

Reply:
I don't know about App, but I think it would need to go in the ReleaseHandler or some container that holds the component.

Reply:
Yeah that makes sense. I don't think it's 100% _un_related to navigation -- as the whole logged-in UI just comes up together as a unit, including the badge -- but it REALLY doesn't belong in the Pure Component.